R/run_match_prioritize.R

```diff
@@ -41,32 +44,32 @@ run_match_prioritize <- function(config) {
 
   # load data----
   ## load manually matched files----
-  list_matched_manual <- list.files(path = dir_matched, pattern = "^matched_lbk_.*_manual[.]csv$")
+  list_matched_manual <- list.files(path = output_matched_loanbooks_dir, pattern = "^matched_lbk_.*_manual[.]csv$")
 
   if (length(list_matched_manual) == 0) {
     cli::cli_abort(c(
       "x" = "No manually matched loan book csvs were found.",
-      "i" = "No files matching the pattern {.code ^matched_lbk_.*_manual[.]csv$} were found in {.path {dir_matched}}. Have you done the manual matching process and named the edited CSVs properly?",
-      "i" = "If {.path {dir_matched}} is not the correct directory, check the {.val dir_matched} parameter set in your {.file config.yml}."
+      "i" = "No files matching the pattern {.code ^matched_lbk_.*_manual[.]csv$} were found in {.path {output_matched_loanbooks_dir}}. Have you done the manual matching process and named the edited CSVs properly?",
+      "i" = "If {.path {output_matched_loanbooks_dir}} is not the correct directory, check the {.val dir_input} parameter set in your {.file config.yml}."
```

Comment:
seems like the directory that has matched loanbooks is in the output directory, not the input directory

Reply:
that is correct and expected. They are the output of one of the steps covered by these functions, which falls into our definition of outputs more broadly

Reply:
```suggestion
      "i" = "If {.path {output_matched_loanbooks_dir}} is not the correct directory, check the {.val dir_output} parameter set in your {.file config.yml}."
```
then this error message should reference the output dir instead of the input dir

Reply:
wouldn't it make sense to do this instead?
```
stop_if_no_files_found(list_matched_manual, output_matched_loanbooks_dir, "dir_output", "manually matched loan book CSVs matching the pattern {.code ^matched_lbk_.*_manual[.]csv$}")
```

Reply:
even better 😃